corehq/toggles.py

```diff
@@ -1475,7 +1475,7 @@ def _commtrackify(domain_name, toggle_is_enabled):
 
 FILTERED_BULK_USER_DOWNLOAD = StaticToggle(
     'filtered_bulk_user_download',
```

Comment:
Any reason that this is not released, it looks like a fairly well functioning feature. (Understand that that's an out of scope for this PR, just curious)

Reply:
I don't know, I agree it seems useful and I don't think it's risky. @dimagi/product do you have thoughts on releasing this more broadly?

Reply:
Can you enter Group name in the Search by Username field?

Reply:
No, it only searches user's first name, last name, and username.